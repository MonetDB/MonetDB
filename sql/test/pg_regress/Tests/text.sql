--
-- TEXT
--

SELECT cast ('this is a text string' AS text) = cast('this is a text string' AS text) AS "true";

SELECT cast ('this is a text string' AS text) = cast('this is a text strin' AS text) AS "false";

CREATE TABLE TEXT_TBL (f1 text);

INSERT INTO TEXT_TBL VALUES ('doh!');
INSERT INTO TEXT_TBL VALUES ('hi de ho neighbor');

SELECT '' AS two, * FROM TEXT_TBL;

-- cleanup
DROP TABLE TEXT_TBL;
