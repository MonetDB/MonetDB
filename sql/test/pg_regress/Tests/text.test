query I rowsort
SELECT cast ('this is a text string' AS text) = cast('this is a text string' AS text) AS "true"
----
1

query I rowsort
SELECT cast ('this is a text string' AS text) = cast('this is a text strin' AS text) AS "false"
----
0

statement ok
CREATE TABLE TEXT_TBL (f1 text)

statement ok
INSERT INTO TEXT_TBL VALUES ('doh!')

statement ok
INSERT INTO TEXT_TBL VALUES ('hi de ho neighbor')

query TT rowsort
SELECT '' AS two, * FROM TEXT_TBL
----
(empty)
doh!
(empty)
hi de ho neighbor

statement ok
DROP TABLE TEXT_TBL

