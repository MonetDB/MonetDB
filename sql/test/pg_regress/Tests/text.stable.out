stdout of test 'text` in directory 'sql/test/pg_regress` itself:


# 22:26:27 >  
# 22:26:27 >   mserver5 "--config=/export/scratch/stripe/fabian/monetdb/current/program-i86pc/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5:/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/lib:/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/bin" --set "gdk_dbfarm=/export/scratch/stripe/fabian/monetdb/current/mtest-pegasus.ins.cwi.nl/sql/dbfarm"    --set mapi_open=true --set xrpc_open=true --set mapi_port=35864 --set xrpc_port=45235 --set monet_prompt= --trace  "--dbname=mTests_src_test_pg_regress" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 22:26:27 >  

# MonetDB server v5.13.0, based on kernel v1.31.0
# Serving database 'mTests_src_test_pg_regress', using 1 thread
# Compiled for x86_64-pc-solaris2.11/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2009 MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://pegasus.ins.cwi.nl:35864/
# MonetDB/SQL module v2.31.0 loaded

# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 11_times.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_querylog.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 19_cluster.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_functions.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 24_zorder.sql
# loading sql script: 25_debug.sql
# loading sql script: 26_sysmon.sql
# loading sql script: 39_analytics.sql
# loading sql script: 40_geom.sql
# loading sql script: 40_json.sql
# loading sql script: 41_md5sum.sql
# loading sql script: 45_uuid.sql
# loading sql script: 46_gsl.sql
# loading sql script: 75_storagemodel.sql
# loading sql script: 80_statistics.sql
# loading sql script: 80_udf.sql
# loading sql script: 90_generator.sql
# loading sql script: 99_system.sql

# 17:59:44 >  
# 17:59:44 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-5665" "--port=31854"
# 17:59:44 >  

#SELECT cast ('this is a text string' AS text) = cast('this is a text string' AS text) AS "true";
% . # table_name
% true # name
% boolean # type
% 5 # length
[ true	]
#SELECT cast ('this is a text string' AS text) = cast('this is a text strin' AS text) AS "false";
% . # table_name
% false # name
% boolean # type
% 5 # length
[ false	]
#CREATE TABLE TEXT_TBL (f1 text);
[ 1	]
[ 1	]
#SELECT '' AS two, * FROM TEXT_TBL;
% .,	sys.text_tbl # table_name
% two,	f1 # name
% char,	clob # type
% 0,	17 # length
[ "",	"doh!"	]
[ "",	"hi de ho neighbor"	]

# 21:59:24 >  
# 21:59:24 >  Done.
# 21:59:24 >  

